MetadataDevelopment/ImplementationChecks/implementationCheckEBML_micro.xsl

```diff
@@ -418,6 +419,12 @@
                         <xsl:with-param name="x" select="mmt:MicroMediaTrace/mmt:b[@n='Segment']/mmt:b[@n='Cluster']/mmt:b[@n='SimpleBlock']/mmt:b[@parser='FFV1']/mmt:d[@n='colorspace_type']"/>
                         <xsl:with-param name="list">0 1</xsl:with-param>
                       </xsl:call-template>
+                      <xsl:call-template name="data_is_in_list">
+                        <xsl:with-param name="icid">FFV1-VALID-VERSION</xsl:with-param>
+                        <xsl:with-param name="version">1</xsl:with-param>
+                        <xsl:with-param name="x" select="$ffv1version"/>
+                        <xsl:with-param name="list">0 1 3.4</xsl:with-param>
```

Comment:
All versions >=3.4 and <4 is expected to be compatible with 3.4, and our parser (and any decoder) should be able to parse it.

"(...) each new micro-version after this first stable variant is compatible with the previous micro-version: decoders SHOULD NOT reject a file due to an unknown micro-version equal or above the micro-version considered as stable."